src/app/lib/device/chromecast.js

```diff
@@ -24,6 +24,7 @@
         play: function (streamModel) {
             var self = this;
             var subtitle = streamModel.get('subFile');
+            var title = streamModel.get('title').substring(0,50);
```

Comment:
Meaning this won't work on a smaller screen, and on a bigger screen there is room left for the title.